# Research

I am currently a member of the [Novembre Lab](jnpopgen.org) at UChicago. I am interested in studying spatial processes in evolution and impacts on human disease traits. My current research focuses on the impact of geographic sampling and dispersal on rare variant ascertainment and models for the spatial spread of adaptive alleles. 

I previously worked in the areas of retroviral evolution and bioinformatics as a research assistant in the [GW Computational Biology Institute](https://cbi.gwu.edu/). I also completed research internships at the following institutions as an undergraduate student: [Memorial Sloan-Kettering Cancer Center Department of Biostatistics (QSURE Fellowship)](https://www.mskcc.org/departments/epidemiology-biostatistics/quantitative-sciences-summer-undergraduate-research-experience-qsure), [UTHealth School of Public Health Dept. of Biostatistics (CPRIT Fellowship)](https://sph.uth.edu/research/opportunities/cprit-undergraduate/), and [University of Pennsylvania Department of Computer and Information Science](https://www.cis.upenn.edu/).

All publications are listed on my [Google Scholar](https://scholar.google.com/citations?user=g0daNV8AAAAJ&hl=en) and [ORCID](https://orcid.org/0000-0002-1062-1228).

## Spatial processes in population genetics

Recent projects include:
* A review of population genetic models for the spread of adaptive alleles in the context of SARS-CoV-2 variant evolution ([published in PLoS Genetics](https://journals.plos.org/plosgenetics/article/comments?id=10.1371/journal.pgen.1010391))
* Modeling the spatial distribution of rare deleterious alleles (presented at the 2022 [Evolution Meetings](https://www.evolutionmeetings.org/) and accepted for presentation at [ASHG 2022](https://www.ashg.org/meetings/2022-annual-meeting/registration/?gclid=Cj0KCQjwyt-ZBhCNARIsAKH1176XNx6HYtN4fxuxaKl_tHkCL64McCNh7aOM-1Mvi6ajJzCkZvWmvQIaAtWYEALw_wcB))

More information coming soon!

## Retroviral phylodynamics & human endogenous retroviruses (undergraduate research)

As a member of the Crandall lab in the GW Computational Biology Institute, I contributed to several research projects in the area of retroviral evolution,  including: 
* Conducting a phylodynamic analysis of HIV epidemics in Washington, D.C. (published in [Frontiers in Microbiology](https://www.frontiersin.org/articles/10.3389/fmicb.2019.00369/full))
* Developing a next generation sequencing pipeline for the analysis of intra-host viral genetic variation (HAPHPIPE; published in [Molecular Biology and Evolution](https://academic.oup.com/mbe/article/38/4/1677/6050845) and available on [Bioconda](https://bioconda.github.io/recipes/haphpipe/README.html))
* Characterizing locus-specific human endogenous retrovirus (HERV) expression in matched tumor-normal RNA-Seq data from The Cancer Genome Atlas (published in [Cancer Research](https://aacrjournals.org/cancerres/article/81/13/3449/666868/Locus-Specific-Characterization-of-Human))
